packages/ui/src/memberships/components/ProfileComponent.tsx

```diff
@@ -18,7 +21,23 @@ export function ProfileComponent() {
       <MemberBalance>
         <BalanceTitle>Total Balance</BalanceTitle>
         <TotalBalance>
-          <TotalTokenValue value={total} />
+          <TokenDetail>
+            <TotalTokenValue value={total} />
+            <StyledTooltip
+              tooltipLinkText="Learn how to earn JOY’s"
+              tooltipLinkURL="https://www.joystream.org/token/"
```

Comment:
As I mentioned in the last review I think this link should become:
```suggestion
              tooltipLinkURL="https://www.joystream.org/token#earn"
```
Even though this id has not been added yet.